megamek/src/megamek/common/MoveStep.java

```diff
@@ -475,6 +473,10 @@ public Coords getTargetPosition() {
     }
 
     public TreeMap<Integer, Vector<Integer>> getLaunched() {
```

Comment:
Does the caller add to this? Or could we return an empty map to them via Collections.emptyMap?

Reply:
I think we better initialize it anyway, out of an abundance of caution. That function is invoked very infrequently, and in places where the minute performance hit of checking if the underlying var has been initialized doesn't matter (user interactions, movement processing on server).